docs/quickstart.rst

```diff
@@ -163,7 +244,13 @@ following.
 
 
 You should see two tests, one that tests the default greeting, and one that
-tests that we can set a custom greeting.  You can run tests using the
+tests that we can set a custom greeting.
+
+Note that both test functions accept a ``chain`` argument. This "chain" is actually py.test fixture, provided by the populus pytest plugin.
```

Comment:
It might be nice if the words *"py.test fixture"* were a link to the pytest docs for fixtures.